core/domain/email_subscription_services_test.py

```diff
@@ -62,8 +65,9 @@ def setUp(self) -> None:
         self.exploration = self.save_new_default_exploration(
             'A', self.editor_id, title='Title')
 
-        self.can_send_subscription_email_ctx = self.swap(
-            feconf, 'CAN_SEND_TRANSACTIONAL_EMAILS', True)
+        self.can_send_subscription_email_ctx = (
```

Comment:
I don't think you should patch non-unit-test methods (like setUp()). Is this a standard pattern in the Python documentation?